src/server.h

```diff
@@ -76,7 +76,7 @@ class ServicePort : public std::enable_shared_from_this<ServicePort>
 class ServiceManager
 {
 	public:
-		ServiceManager() = default;
+		ServiceManager(boost::asio::io_service& service);
```

Comment:
It's a single like, just inline it here :)

Reply:
Not sure I understand you here.

Reply:
The implementation of this constructor is a single line, it should be inlined into the header:

```
 		ServiceManager(boost::asio::io_service& service) : io_service{service} {}
```